webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -606,13 +606,21 @@
 
                 <div id="rawdump" class="ui-tabs-hide">
                     <div class="code">{{ raw_stackwalker_output }}</div>
-                    {% if request.user.has_perm('crashstats.view_rawdump') %}
+
                     <h3>Download the Raw Dump</h3>
+                    {% if request.user.has_perm('crashstats.view_rawdump') %}
                     {% for url in raw_dump_urls %}
                     <p><a href="{{ url }}">{{ url }}</a></p>
                     {% endfor %}
                     {% else %}
-                    <p>You need to be signed in to be able to download raw dumps.</p>
+                    <p>You need to be signed in to download raw dumps.</p>
+                    {% endif %}
+
+                    <h3>View the Unredacted Crash</h3>
+                    {% if request.user.has_perm('crashstats.view_rawdump') %}
+                    <p><a href="{{ '../../api/UnredactedCrash/?crash_id=' + report.uuid }}">{{ '/api/UnredactedCrash/?crash_id=' + report.uuid }}</a></p>
```

Comment:
You also probably want to open that in another tab or window, since this link won't trigger a download. Add ``target="_blank"`` to the ``<a>`` tag for example.